### PR TITLE
Add Open source office chapter with some basic text

### PR DESCRIPTION
**Pull request by DunderRoffe**
The point of this PR is to create a first brief of the OSO chapter.
It also adds a peer-review-checklist

Questions:
* how should/does this relate to the "licensing" part in SWF-blueprint?
* should it be called something else?
* is this enough as a first thing ?
* should something be moved to the SWF-blueprint?

### CODE REVIEW

File: docs/chapters/oso/checklist-for-peer-reviewing-repository-requirements.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Checklist for peer reviewing repository requirements
+====================================================
+
+#. Make sure that the following files are present in the root of the source code
+   repository:
+
+   * LICENSE: Stating the license of the source code.
+   * README.md: Containing information on where to find the documentation of the
+     product. There should also be a notice of where the code is maintained and
+     a that the license can be found in the LICENSE file.
```
Comment (thenor) [inline docs/chapters/oso/checklist-for-peer-reviewing-repository-requirements.rst]: Missing word after 'a'.

**Review by thenor — CHANGES_REQUESTED** [review]
Just a small comment in the checklist text.

File: docs/index.rst [hunk 2]
```diff
@@ -17,6 +17,7 @@ The Software Factory documentation is the central space where we collect all use
     chapters/ci-and-cd/index
     chapters/sdk/index
     chapters/sde/index
+    chapters/oso/index
```
Comment (tobsan) [inline docs/index.rst:20]: Not sure about the path name here. OSO is not as obvious as an acronym as SDK for example. But I don't have a strong opinion about it anyway.

**Review by tobsan — APPROVED** [review]
Looks good given that you fix what @thenor asked for.

File: docs/index.rst (hunk 2 above)
Comment (DunderRoffe) [reply docs/index.rst:20]: OSO is the acronym used for the open source office (although so far mostly internally) so I think it makes sense to keep using it.

Comment (breakreturn) [reply docs/index.rst:20]: Maybe it's nicer to spell it out? "chapters/opensourceoffice/index" ?

**Review by thenor — APPROVED** [review]

**Comment by tobsan** [comment]
@DunderRoffe is something happening with this PR?

**Comment by DunderRoffe** [comment]
I am waiting for response. There are ongoing work to improve this content but I feel that this is a good start

**Comment by breakreturn** [comment]
I don't think the Baseline SWF should contain a new description of the OSO which will be in the Blueprint as well for example. I also think that this is overlapping with the text which is currently being created elsewhere. I don't think it's very clear how this PR would relate to that and I'd rather see one complete attempt to get this fairly complete in one place so it's possible to review the whole thing. 

So I'd vote for closing this and incorporate the content in that other content and make it one complete set of texts.

**Comment by DunderRoffe** [comment]
@breakreturn the other content you are referring to is already based on this. In my mind it is the next version of what this attempts to create as a start.

Also when attempting to put this as a PR to the blueprint it was rejected by you because it should be specific for the baseline and not the in the blueprint.

**Comment by breakreturn** [comment]
@DunderRoffe Yeah OK. I think that both this and the other content has both Blueprint and Baseline SWF content. My point is more that it's difficult and maybe wasteful to do a thorough review of this when I know it's WiP somewhere to make a lot of that review feedback moot. 

But given that we agree to update this according to the other material etc, then I'm not totally opposed to merging this, I'm just not sure I see the point. But If you and others see a point then go ahead, that's fine by me in this case.

**Comment by tobsan** [comment]
It would be nice to get a resolution to this. Imho if this is blocking further development on the docs for this topic we should just pull it in. I think it is enough as a first step.

**Comment by jeremiah** [comment]
There is some more material internally on our 'corporate' wiki. I'll post the link in hipchat for review.

**Pull request closed without merging**